custom_conf.py

```diff
@@ -113,42 +116,61 @@
 
 # Set up redirects (https://documatt.gitlab.io/sphinx-reredirects/usage.html)
 # For example: 'explanation/old-name.html': '../how-to/prettify.html',
-
+# NOTE: If this variable is not defined, set to None or the dictionary is empty
+# the sphinx_reredirects extensions will be disabled.
 redirects = {}
 
 ############################################################
 ### Link checker exceptions
 ############################################################
 
 # Links to ignore when checking links
-
 linkcheck_ignore = [
     'http://127.0.0.1:8000'
     ]
 
 # Pages on which to ignore anchors
 # (This list will be appended to linkcheck_anchors_ignore_for_url)
-
-custom_linkcheck_anchors_ignore_for_url = [
-    ]
+custom_linkcheck_anchors_ignore_for_url = []
 
 ############################################################
 ### Additions to default configuration
 ############################################################
 
 ## The following settings are appended to the default configuration.
 ## Use them to extend the default functionality.
+# NOTE: Remove this variable to disable the my MyST parser extensions. Please
```

Comment:
```suggestion

# NOTE: Remove this variable to disable the MyST parser extensions. Please
```

Reply:
Again, what do you mean with "consult with upstream"?

Reply:
Note: I removed this remark with c860cca8def2837cf743eb8548039cf0ba5e5e0f